x/params/types/consensus_params.go

```diff
@@ -1,27 +1,26 @@
-package keeper
+package types
 
 import (
 	abci "github.com/tendermint/tendermint/abci/types"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
 	"github.com/cosmos/cosmos-sdk/baseapp"
-	"github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
 // ConsensusParamsKeyTable returns an x/params module keyTable to be used in
 // the BaseApp's ParamStore. The KeyTable registers the types along with the
 // standard validation functions. Applications can choose to adopt this KeyTable
 // or provider their own when the existing validation functions do not suite their
 // needs.
-func ConsensusParamsKeyTable() types.KeyTable {
-	return types.NewKeyTable(
-		types.NewParamSetPair(
+func ConsensusParamsKeyTable() KeyTable {
```

Comment:
Could we add an API breaking changelog entry for this guy?

Reply:
yes, on it